code/src/index.css

```diff
@@ -1,13 +1,416 @@
+
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  /* font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+    sans-serif; */
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-family: "Helvetica Neue", Sans-Serif;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+main{
+  display: flex;
+  height: 100vh;
+  flex-direction: column;
+  background-image: url('https://images.unsplash.com/photo-1559251606-c623743a6d76?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80');
+  background-size: cover;
+  background-position-y: start;
+  background-position-x: center;
+  background-repeat: no-repeat;
+  color: #fff;
+  --red: #e16561;
+}
+
+input,
+select,
+input:active,
+select:active {
+	-webkit-appearance: none;
+	-moz-appearance: none;
+	appearance: none;
+}
+
+/* =====FONT STYLES=====*/
+h1{
+  font-size: 60px;
+  font-weight:700;
+  background:  #fff;
+  background-clip: text;
+	-webkit-background-clip: text;
+  color: transparent;
+  text-align: center;
+  margin:0;
+}
+
+h2{
+  line-height: 40px;
+  text-align: center;
+  margin:0;
+}
+h3{
+  margin: 0;
+  margin:0;
+}
+p{
+  margin: 0;
+  margin:0;
+}
+.labelStyle{
+  font-size: 24px;
+  font-weight: 700;
+  align-self: flex-start;
+}
+/* =====PROGRESS BAR STYLES=====*/
+.progress-container{
+  width:100%;
+  height:1rem;
+  background-color: #fff;
+  display:flex;
+  align-items: center;
+  opacity: 0.8;
+  border-radius: 0.5rem;
+}
+
+.progress-value{
+  height:80%;
+  background-color:var(--red);
+  animation: load 3s normal forwards;
+  opacity: 0.8;
+  border-radius: 0.4rem;
+  margin: 0 0.1rem;
+}
+
+@keyframes load {
+  0% {
+    width:0;
+  }
+  100% {
+    width: 1;
+  }
+}
+/* =====CONTAINER STYLES=====*/
+.container{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  width: 300px;
+  height: 100%;
+  margin: auto;
+  gap: 1rem;
+}
+
+/* =====BUTTON STYLES=====*/
+.button {
+  display: flex;
+  align-items: center;
+  padding: 0.5rem;
+	font: 13px ;
+	font-weight: 900;
+	letter-spacing: 2px; 
+	text-transform: uppercase;
+	text-decoration: none;
+	text-align: center;
+  color: var(--red);
+	background-color: #fff;
+	border: 5px solid transparent;
+	transition: .2s ease;
+	border-radius: 50px;
+  align-self: center;
+}
+
+.button:disabled{
+  opacity: 0.2;
+}
+
+.button:hover {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.20), 0 6px 20px 0 rgba(0, 0, 0, 0.20);
+  color: #fff;
+	background-color: var(--red);
+	transform: scale(1.1);
+  cursor: pointer;
+}
```

Comment:
the indentions here is a bit uneven, would look  more clean if they were the same :)